### PR TITLE
feat: change url in notifications to manage-cases (FPLA-1297)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://tools.hmcts.net/jira/browse/FPLA-1297
related: https://github.com/hmcts/cnp-flux-config/pull/3575/files
feature toggle name: expert-ui (turned off on prod, turned on on all other envs)

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+
+@Service
```
Comment (Gogetter) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java:11]: minor: currently only one dependency but we can use `@RequiredArgsConstructor(onConstructor = @__({@Autowired}))`.

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseUrlServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {CaseUrlService.class,})
+@TestPropertySource(properties = {"ccd.ui.base.url=" + CCD_UI_URL, "manage-case.ui.base.url=" + XUI_URL})
+class CaseUrlServiceTest {
+
+    static final String CCD_UI_URL = "http://ccd";
+    static final String XUI_URL = "http://xui";
+
+    private long caseId = RandomUtils.nextLong();
```
Comment (Gogetter) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseUrlServiceTest.java:25]: final?

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProviderTest.java [hunk 3]
```diff
@@ -1,26 +1,40 @@
...
+
+    @MockBean
+    CaseUrlService caseUrlService;
+
+    @MockBean
+    FeatureToggleService featureToggleService;
+
+    @BeforeEach
+    void initCaseUrlService() {
+        when(caseUrlService.getCaseUrl(anyLong())).thenAnswer(inv -> caseUrl(inv.getArgument(0).toString()));
```
Comment (Gogetter) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProviderTest.java:38]: could we have `inv` as `invocation`?

File: charts/fpl-case-service/Chart.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.11.1
+version: 1.11.2
```
Comment (Gogetter) [inline charts/fpl-case-service/Chart.yaml:4]: we might have to bump this as #1217 might be merged before this

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java (hunk 1 above)
Comment (tomaszpowroznik) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java:12]: done

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProviderTest.java (hunk 3 above)
Comment (tomaszpowroznik) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProviderTest.java]: done

File: charts/fpl-case-service/Chart.yaml (hunk 4 above)
Comment (tomaszpowroznik) [reply charts/fpl-case-service/Chart.yaml]: done

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseUrlServiceTest.java (hunk 2 above)
Comment (tomaszpowroznik) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseUrlServiceTest.java]: done

**Pull request merged**